weaviate/collections/classes/cluster.py

```diff
@@ -24,23 +24,24 @@ class Stats:
 
 Shards = List[Shard]
 S = TypeVar("S")
+S2 = TypeVar("S2")
```

Comment:
Perhaps naming these `Sh` and `St` might be nicer, wdyt?

Reply:
sure